src/modules/order/1652101349791-order.migration.ts

```diff
@@ -0,0 +1,18 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+import { Migration } from 'medusa-extender';
+
+@Migration()
+export class OrderMigration1652101349791 implements MigrationInterface {
+    name = 'OrderMigration1652101349791';
+    
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        const query = 'ALTER TABLE public."order" ADD COLUMN IF NOT EXISTS "store_id" text; ALTER TABLE public."order" ADD COLUMN IF NOT EXISTS "parent_id" text;';
```

Comment:
would it make more sense to name the new column as `order_id` instead.
Because, order_id mean that the parent in an order whereas `parent_id` could mean that a parent is some entity but you don't know at first glance.

Reply:
It is also missing the foreign key constraint on itself (id) that is nullable

Reply:
yeah I see your point. parent_order_id is fine with me :)